### ISSUE TITLE
Unable to install Pascal Workshop

### ISSUE THREAD

**Issue by kgraaf** [arcanebyte/lisaem#4]
Installing the pascal workshop (v 3.0) fails to succeed. Attempting to erase/format the disk fails and the workshop installer complains that it cannot write to the disk. I've tried the installation with two different copies of Workshop (from Bitsavers and Macintosh Garden), 5M and 10M disks, the CPU throttled to 5Mhz, with 1.5MB and 1MB of RAM. All possible configurations fail with the same error.

Background:

* Compiled Lisaem in an Ubuntu Docker container with a fresh copy of wxWidgets. Compilation succeeded with no obvious errors
* Successfully installed LOS
* Running with a boot ROM created per the procedure documented in another issue

<img width="363" alt="Screenshot_20240130_012241" src="https://github.com/arcanebyte/lisaem/assets/82181542/aa529ba0-496d-4b19-953f-1dbebae552a4">

**Comment by busterswt** [comment]
Thanks for the report. I've more or less confirmed this on a master branch build on macOS 14.0. Hasn't reported any errors, yet, but is taking a long time trying to install on Disk 1.

**Comment by kgraaf** [comment]
Were you able to install the Workshop successfully?

**Comment by busterswt** [comment]
> Were you able to install the Workshop successfully?

No, but it has worked in the past. Will try an older codebase to see if there's a regression.

**Comment by kgraaf** [comment]
I remember being able to install before Ray (RIP) fixed The Bug that caused the linker to blow up. I wonder if that fix somehow broke installing the workshop? Do we know if the linker (and scrollbars in LOS, iirc) and workshop install ever worked at the same time?

**Comment by kgraaf** [comment]
Curious if any progress has been made on this

**Comment by arcanebyte** [comment]
@kgraaf - @TorZidan has reverted some previous profile-related code that might have fixed this issue. I was able to successfully install Workshop 3.0 under the following scenario:

I/O ROM A8
5M ProFile image
Rev H ROM
wxWidgets 3.2.6
Latest master branch (which has pulled in the fixes)

Give that a go and let us know!
